ooni/ui/web/client/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  </head>
+  <body ng-app="app" ng-strict-di ng-cloak>
+
+    <app>
+      Loading...
```

Comment:
Currently we don't have a tagged version of the latest gui. For testing you should build it yourself from the master.

How I do this while developing the web-ui is that I build it using:

```
gulp build --watch --dest /path/to/ooni-probe/ooni/ui/web/client/
```